documentation/source/building.rst

```diff
@@ -62,6 +62,14 @@ Make Variables
 
       Passed to both the compile and execute phases of simulators with two rules, or passed to the single compile and run command for simulators which don't have a distinct compilation stage.
 
+.. make:var:: COCOTB_HDL_TIMEUNIT
+
+      The default time unit that should be assumed for simulation when not specified by modules in the design. If this isn't specified then it is assumed to be ``1ns``. Allowed units are ``s``, ``ms``, ``us``, ``ns``, ``ps``, ``fs``.
+
+.. make:var:: COCOTB_HDL_TIMEPRECISION
+
+      The default time precision that should be assumed for simulation when not specified by modules in the design. If this isn't specified then it is assumed to be ``1ps``. Allowed units are ``s``, ``ms``, ``us``, ``ns``, ``ps``, ``fs``.
```

Comment:
There's one more restriction: allowed values (orders of magnitude) are only 1, 10, or 100, for both timeunit and timeprecision. 

You could add an example like `timescale COCOTB_HDL_TIMEUNIT/COCOTB_HDL_TIMEPRECISION this should make it rather obvious what is meant to a HDL engineer, even if they have not heard the naming as it is used in the Verilog spec.